src/poetry/core/packages/utils/link.py

```diff
@@ -266,17 +266,6 @@ def is_egg(self) -> bool:
     def is_sdist(self) -> bool:
         return self.ext in {".tar.bz2", ".tar.gz", ".zip"}
 
-    @cached_property
-    def is_artifact(self) -> bool:
```

Comment:
why was this removed?

Reply:
SIM103 didn't like the way that it was written: but since the code is unused it is cleaner to remove it altogether than to rearrange it